### ISSUE TITLE
Build ARM Image for the Operator

### ISSUE THREAD

**Issue by mms-gianni** [kubero-dev/kubero#128]
How to check : 

```
kubectl get pods --all-namespaces | grep kubero-operator-controller-manage
```

Results in following error: 
```
kubero-operator-system   kubero-operator-controller-manager-XXXXXXXXX   0/2     ErrImagePull        0          3m8s
```

**Comment by mms-gianni** [comment]
The ARM images are available now.

**Comment by mms-gianni** [comment]
I still had no possibility to verify these images. Is anyone with an M1/ARM here to test it?

**Comment by Givemeurcookies** [comment]
I tested the operator with the [Hetzner Terraform k3 cluster](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner), the cluster I set up only uses ARM servers (CAX) and the operator seems to work. I will be testing over the next few days.

Kubero also seems to work pretty well out of the box with the Hetzner k3s terraform script, the dependencies seems to be almost the same and the Hetzner load balancer didn't require any config either. The only thing required is to point the DNS records towards the Hetzner load balancer public IP.

Other than that, a mention about ARM; the CLI seems to be lacking "developer verification" and the system is unable to scan it for malware so it is a bit of a pain to get working with a M1 Max.

**Comment by mms-gianni** [comment]
I finally have a Raspberry Pi to test it with k3s and arm images. 

I am still investigating how to add a developer verification. So far, I've found this issue that describes the Code signing. https://github.com/golang/go/issues/42684 . 

TLDR;
To sign the binary on a Mac M1 just run `codesign -s - /usr/local/bin/kubero`. But this must be run on the local computer after installation.
